karma.conf.js

```diff
@@ -59,7 +57,7 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['Chrome', 'Firefox'],
+    browsers: ["Firefox"],
```

Comment:
Why the browser removal?

Reply:
Only firefox is used in https://github.com/sphinx-doc/sphinx/blob/b22134dcd35c9cffd5c8fe5a3f5224fcb3d3741f/package.json#L4

This was attempting to simplify testing -- if better to drop can do.

Reply:
Ah, neat. Then, this seems fine by me. I think this should get reintroduced with `--browsers Firefox,Chrome` in the future, but... that's a reasonable follow up PR. :)